carta/image.py

```diff
@@ -65,8 +69,13 @@ def new(cls, session, path, hdu, append):
         """
         path = session.resolve_file_path(path)
         directory, file_name = posixpath.split(path)
-        image_id = session.call_action("appendFile" if append else "openFile", directory, file_name, hdu, return_path="frameInfo.fileId")
-
+        command = "appendFile" if append else "openFile"
+        if complex is False:
+            image_arithmetic = False
+        elif complex is True:
+            image_arithmetic = "true"
```

Comment:
This should be `True`, not `"true"` (I missed this last time).

Reply:
Actually, instead of explicitly comparing to `True` and `False`, this would be more cleanly expressed as:
```
if complex:
    (...)
else:
    (....)
```

Reply:
Done in 8e40b99. Thanks a lot.